pkg/stanza/operator/input/windows/subscription.go

```diff
@@ -106,8 +107,9 @@ func (s *Subscription) createFlags(startAt string, bookmark Bookmark) uint32 {
 }
 
 // NewSubscription will create a new subscription with an empty handle.
-func NewSubscription() Subscription {
+func NewSubscription(server string) Subscription {
```

Comment:
Two function will be self documenting regarding the `server` parameter: `NewRemoteSubscription(server string)` and `NewLocalSubscription()`

Reply:
Added in both new ways to call a subscription